packages/forma-36-react-components/.size-limit

```diff
@@ -3,7 +3,7 @@
     "path": "dist/index.js",
     "name": "gzipped js",
     "gzip": false,
-    "limit": "390 KB",
+    "limit": "401 KB",
```

Comment:
What's causing such a size increase?

Reply:
I assume this is somehow related to creating a new package?